File: backend/src/main/java/kerosene/v05/contracts/UserDTO.java
package kerosene.v05.contracts;

public interface UserDTO {


    String getUsername();
    String getPassphrase();
    String getTotpSecret();
    String getTotpCode();

    void setUsername(String username);
    void setPassphrase(String passphrase);
    void setTotpSecret(String totpSecret);
    void setTotpCode(String totpCode);



}

File: frontend/lib/features/authentication/domain/entities/UserDTO.dart
class User {
  String username = '';
  String passphrase = '';
  String totpSecret = '';
  String totpCode = '';

  static final User instance = User._internal();
  User._internal();

  String getUsername() => username;
  String getPassphrase() => passphrase;
  String getTotpSecret() => totpSecret;
  String getTotpCode() => totpCode;

  void setUsername(String u) => username = u;
  void setPassphrase(String p) => passphrase = p;
  void setTotpSecret(String t) => totpSecret = t;
  void setTotpCode(String c) => totpCode = c;
}
